src/main/kotlin/solve/settings/visualization/fields/view/VisualizationSettingsLayerCell.kt

```diff
@@ -53,12 +57,42 @@ class VisualizationSettingsLayerCell(
         scene.changeLayerIndex(droppedItemInfo.item, droppedItemGroupRenderIndex)
     }
 
-    override fun createItemCellGraphic(item: LayerSettings): Node = hbox {
+    private fun createItemCellGraphicSnapshot(item: LayerSettings): Node = hbox(5) {
```

Comment:
duplicates `createItemCellGraphic`, I can not understand why we have plane specific logic in one case and don't have in other

Reply:
That function was necessary beacuse of the additional logic that included bindings for editing buttons.